src/gui/widgets/scrollbar.hpp

```diff
@@ -158,10 +159,14 @@ class scrollbar_base : public styled_widget
 	{
 		return step_size_;
 	}
+
 	void set_step_size(const unsigned step_size)
 	{
-		step_size_ = step_size;
-		recalculate();
+		// Step size can't be 0!
+		if(step_size != 0) {
```

Comment:
If step size of zero isn't allowed, how about throwing an exception instead of just ignoring the call?